content/_en/help/verify-your-identity/accepted-state-issued-identification.md

```diff
@@ -4,18 +4,19 @@ title: "Accepted State-Issued Identification "
 category: verify-your-identity
 order: 2
 ---
-At this time, only the following state-issued identification is accepted: 
+You cannot verify your identity on login.gov without a state-issued ID. We’re currently working to add more ways to verify your identity. At this time, only the following state-issued identification is accepted: 
```

Comment:
What do you think of phrasing this first sentence without a double negative? Maybe "Currently, you need a state-issued ID to verify your identity on login.gov"